docs/_documentations/che-setupregistries.md

```diff
@@ -8,26 +8,37 @@ permalink: che-setupregistries.html
 type: document
 ---
 
-# Adding image registries in Codewind in Che
+# About image registries
 
-Ensure you have completed the [prerequisites to installing Codewind on Eclipse Che](./che-installinfo.html#prerequisites).
+An image registry is an online service that stores Docker images. Docker images are either images that you build yourself, or images from third parties that you want to consume, for example, a base image of Node.js or a base image of Java. The image registry is also the place where Kubernetes and OpenShift download from when they deploy the application for ongoing development or testing within each of those environments.
 
-**Note:** If you completed the [Codewind for Eclipse Che prerequisites and installation steps](./che-installinfo.html#installing-che-to-use-with-codewind) for Kubernetes, then you have installed Eclipse Che with self-signed certificates. To proceed, you must either select 'accept the certificates' in your browser or add the certificates to your browser. Ask your cluster administrator for further information. **Caution:** do not use self-signed certificates in production
+Some registries are protected and limit access to authenticated users only. You might use one or multiple image registries each having different credentials. In the following section we'll discuss when you'll need a registry configured within Codewind, and how to store your credentials for use when the registry has been secured. 
+
+## When you need to define an image registry
+You need to enter configuration details for the image registry for any of the following conditions:
+- Building a project with a Dockerfile that pulls from a protected image registry that requires authentication.
+- Working with Appsody projects that pull stack images from a protected image registry that requires authentication.
+- When you are using Codewind remotely
+- When you are using Che to build Codewind style projects
 
-## Prerequisite: Determining if you need an image registry 
-1\. First, determine whether you need to add an image registry. The following scenarios in Codewind require you to specify which image registry is in use:
-- In the following scenario, configure Codewind with an image registry to push the application image to that specific image registry:
+### Determining whether you need an image registry for Che
+You need an image registry for Che if:
+- In the following scenario, you want to configure Codewind with an image registry to push the application image to that specific image registry:
     - Run Codewind on a remote Kubernetes cluster, including Che, to develop a Codewind style project.
```

Comment:
This section is talking about Che specifically so could we remove line 26 and just change 27 to say:

When developing Codewind style projects.

Reply:
Changed to:
You need an image registry for Che if:
- You are developing Codewind style projects.